src/Datadog.Trace/Agent/ZipkinApi.cs

```diff
@@ -0,0 +1,103 @@
+using System;
+using System.Net;
+using System.Threading.Tasks;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.Logging;
+
+namespace Datadog.Trace.Agent
+{
+    internal class ZipkinApi : IApi
+    {
+        private static readonly Vendors.Serilog.ILogger Log = DatadogLogging.For<ZipkinApi>();
+
+        private readonly TracerSettings _settings;
+        private Uri _tracesEndpoint;
+
+        public ZipkinApi(TracerSettings settings)
+        {
+            Log.Debug("Creating new Zipkin Api");
+
+            _settings = settings ?? throw new ArgumentNullException(nameof(settings));
+            _tracesEndpoint = _settings.AgentUri; // User needs to include the proper path.
+        }
+
+        public void SetBaseEndpoint(Uri baseEndpoint)
+        {
+            // TODO: This doesn't make sense for this exporter.
```

Comment:
This is slated to be removed in the datadog branch, so a TODO to remove this is sufficient